src/components/DatePicker/DatePicker.st.css

```diff
@@ -0,0 +1,581 @@
+    CurrentSelectedDayBGColor: color(fallback(value(SelectedDayBGColor), value(DefaultSelectedDayBGColor)));
+    CurrentSelectedDayBorderRadius: fallback(value(SelectedDayBorderRadius),value(DefaultSelectedDayBorderRadius));
+    CurrentDisabledDayColor: color(fallback(value(DisabledDayColor), value(DefaultDisabledDayColor)));
+}
+
+/* The component's css without the overrideStyleParams */
+
+.root {
+}
+
+.root .popoverContent {
+    border-radius: 0;
+}
+
+.calendar {
+    -st-extends: WSRCalendar;
+}
+
+.calendar::month {
+    width: value(calendarWidth);
+    height: value(calendarHeight);
+}
+
+/* calendar header */
+
+.calendar::header {
+    padding: 28px value(sidePadding) 0;
+    background-color: value(CurrentBackgroundColor);
+}
+
+.calendar::header::arrow {
+    color: value(CurrentNavigationArrowsColor);
+    width: 24px;
+    height: 24px;
+}
+.calendar::header::arrow:hover:not(active) {
+    background: none;
+    color: applyOpacity(value(CurrentNavigationArrowsColor), 0.7);
+}
+
+.calendar::header::arrow:active {
+    background: none;
+    color: applyOpacity(value(CurrentNavigationArrowsColor), 0.7);
+}
+
+.calendar::header::caption {
+    font: value(CurrentMonthAndYearCaptionFont);
+    font-size: 16px;
+    color: value(CurrentMonthAndYearCaptionColor);
+}
+
+/******************************************************/
+
+/* calendar Weekdays */
+
+.calendar::weekdays {
+    padding: 0 value(sidePadding);
+    background-color: value(CurrentBackgroundColor);
+}
+
+.calendar::weekday {
+    padding: 20px 0 16px;
+    font: value(CurrentWeekdayFont);
+    font-size: 14px;
+    color: value(CurrentWeekdayTextColor);
+}
+
+/******************************************************/
+
+/* calendar body */
+
+.calendar::body {
+    padding: 0 value(sidePadding) 20px;
+    background-color: value(CurrentBackgroundColor);
+}
+
+.calendar::day {
+    font: value(CurrentDayFont);
+    font-size: 14px;
+    color: value(CurrentDayColor);
+    width: 34px;
+    height: 34px;
+}
+
+.calendar::day:hover {
+    color: color(value(DeafultDayColor));
+    background-color: value(CurrentDayBGColorHover);
+}
+
+.calendar::day:today:not(:selected):not(:hover) {
+    background: none;
+    color: value(CurrentSelectedDayBGColor);
+    box-shadow: none;
+}
+
+.calendar::day:today:not(:selected):hover {
+    color: value(CurrentSelectedDayBGColor);
+    background-color: value(CurrentDayBGColorHover);
+}
+
+.calendar::day:today:not(:selected):not(:hover):focus {
+    box-shadow: none;
+}
+
+.calendar::day:selected:not(:outside) {
+    color: value(CurrentSelectedDayColor);
+}
+
+.calendar::day:outside, .calendar::day:disabled {
+    color: value(CurrentDisabledDayColor);
+}
+
+/* todo: need to check if it's really override the viewer focus ring and if we should remove it..  */
+.calendar::day:focus::after {
+    border-radius: value(CurrentSelectedDayBorderRadius);
+    box-shadow: 0 0 0 1px #ffffff, 0 0 0 3px #116dff !important;
```

Comment:
You can see the correct implementation (which is very similar) [here](https://github.com/wix/wix-ui-tpa/blob/2eb09078873af6f8164f0123626139b62c7a7bb7/src/components/RadioButton/RadioButton.st.css#L96).

It would be nice to move it to a mixin in a common file such as palette.st.css